BigriverBookstore-apple/Sources/Model/Book.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Book.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Spine
+
+class Book: Resource {
+    var title: String?
+    var isbn: NSNumber?
```

Comment:
@c-Viorel Can we use Int or Double (or any Swift type) to avoid using ObjC NSNumber?

Reply:
No, Spine frameworks want NSNumber